src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -109,41 +525,176 @@ struct FixedKVInternalNode
 	    ? &delta_buffer : nullptr;
   }
 
-  CachedExtentRef duplicate_for_write() override {
+  CachedExtentRef get_mutable_replica(Transaction &t) override {
+    LOG_PREFIX(FixedKVInternalNode::get_mutable_replica);
     assert(delta_buffer.empty());
-    return CachedExtentRef(new node_type_t(*this));
-  };
+    auto ext = new node_type_t(*this);
+    auto tid = t.get_trans_id();
+    ext->touched_by = tid;
```

Comment:
This seems to be a duplicate of the update in CachedExtent::get_mutable_replica(), see my other comment.